src/Uno.UWP/Devices/Lights/Lamp.Android.cs

```diff
@@ -0,0 +1,152 @@
+﻿#if __ANDROID__
+using System;
+using System.Linq;
+using Android.Graphics;
+using Android.Hardware.Camera2;
+using Android.OS;
+using Android.App;
+using Android.Content;
+using Android.Content.PM;
+
+namespace Windows.Devices.Lights
+{
+	public partial class Lamp
+	{
+		private readonly object _lock = new object();
+
+		private readonly string _defaultCameraId;
+		private CameraManager _cameraManager;
+
+#pragma warning disable CS0618
```

Comment:
Add a comment here explaining why the deprecated API is used

Reply:
Done 👍